src/DynamicEvaluator/Application/Checker/BrokenPhpFilesChecker/Dto/ViolationDto.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+declare(strict_types=1);
+
+namespace DynamicEvaluator\Application\Checker\BrokenPhpFilesChecker\Dto;
+
+use Upgrade\Application\Dto\ViolationDtoInterface;
+
+class ViolationDto implements ViolationDtoInterface
```

Comment:
I would rename this DTO to something specific. it's so abstract name class for the package. - just recommendation

Reply:
This violation has a limited scope otherwise violation Interface should be used